registration-processor/pre-processor/registration-processor-supervisor-validator-stage/src/test/java/io/mosip/registration/processor/app/SupervisorValidatorProcessorTest.java

```diff
@@ -164,35 +166,44 @@ public void testisValidOperatorSuccess() throws Exception {
 	public void IOExceptionTest() throws Exception {
 
 		Mockito.doThrow(new IOException()).when(supervisorValidator).validate(anyString(), any(), any());
-		assertEquals(false, supervisorValidationProcessor.process(dto, stageName).getIsValid());
-		assertEquals(true, supervisorValidationProcessor.process(dto, stageName).getInternalError());
+		Mockito.when(registrationStatusMapperUtil
+				.getStatusCode(RegistrationExceptionTypeCode.IOEXCEPTION)).thenReturn("ERROR");
+		MessageDTO object = supervisorValidationProcessor.process(dto, stageName);
+		assertFalse(object.getIsValid());
+		assertTrue(object.getInternalError());
 	}
 
 	@Test
 	public void ValidationFailedExceptionTest() throws Exception {
 
 		Mockito.doThrow(new ValidationFailedException("id", "message")).when(supervisorValidator).validate(anyString(),
 				any(), any());
-		assertEquals(false, supervisorValidationProcessor.process(dto, stageName).getIsValid());
-		assertEquals(false, supervisorValidationProcessor.process(dto, stageName).getInternalError());
+		MessageDTO object = supervisorValidationProcessor.process(dto, stageName);
+		assertFalse(object.getIsValid());
+		assertFalse(object.getInternalError());
 	}
 
 	@Test
 	public void apiResourceExceptionTest() throws Exception {
 
 		Mockito.doThrow(new ApisResourceAccessException("")).when(supervisorValidator).validate(anyString(), any(),
 				any());
-
-		assertEquals(false, supervisorValidationProcessor.process(dto, stageName).getIsValid());
-		assertEquals(true, supervisorValidationProcessor.process(dto, stageName).getInternalError());
+		Mockito.when(registrationStatusMapperUtil
+				.getStatusCode(RegistrationExceptionTypeCode.BASE_CHECKED_EXCEPTION)).thenReturn("ERROR");
+		MessageDTO object = supervisorValidationProcessor.process(dto, stageName);
+		assertFalse(object.getIsValid());
```

Comment:
this shoud be isValue true and isInternalError true

Reply:
handled this scenario,.,